Framework/Utils/src/DPLGatherer.cxx

```diff
@@ -42,9 +42,9 @@ o2f::DataProcessorSpec defineGatherer(std::string devName, o2f::Inputs usrInputs
                  // Retrieving message size from API
                  auto msgSize = (o2::header::get<o2::header::DataHeader*>(itInputs.header))->payloadSize;
                  // Allocating new chunk
-                 auto fwdMsg = ctx.outputs().newChunk((*outputPtr), msgSize);
+                 auto& fwdMsg = ctx.outputs().newChunk((*outputPtr), msgSize);
```

Comment:
(also in other places) Isn't it better to use an `observer_ptr` so that there is no risk of coping this?

Reply:
this is indeed a problem which needs to be solved. In line with outer `make` specializations, returning a reference is more consistent. I was thinking about how to catch this case, but did not converge. I'm adding an item to the tick list.

Reply:
Yeah, I guess I do not like what we do in make in general... :-) Another possibility would be to use `std::ref` / `std::reference_wrapper`, even if that would mean you need to use `operator()` to invoke methods on it (because . cannot be overloaded)...

Reply:
`std::ref` is probably a good option, but I tend to not change this in this PR but consolidate this later. Will give it a try.

Reply:
see https://github.com/AliceO2Group/AliceO2/pull/1846#issuecomment-480246198 for a summary